ci/mg/build.sh

```diff
@@ -51,8 +51,8 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       protobuf >=3.4.1,<4.0.0 \
       libclang \
       nccl>=2.4 \
-      dask=2.3.0 \
-      distributed=2.3.0 \
+      dask=2.12* \
```

Comment:
WOW. Glad this got revisited even if it's not actively important today to have the right dask here.

Reply:
Probably not critical, but would use `dask=2.12.*`. Though it's unlikely we will get to `2.120` without making some kind of breaking change first. 😉

Reply:
@cjnolet @jakirkham please match version pinning that cudf uses here and elsewhere 

https://github.com/rapidsai/cudf/pull/4412/files#diff-f60d66f138bb22df65514ede91c1a51dR37

I'm not a fan of conflicting pinning schemes